mkdocs-base.yml

```diff
@@ -119,204 +118,352 @@ extra:
+          - install-pmm/install-pmm-server/virtual/vmware.md
+          - install-pmm/install-pmm-server/virtual/virtualbox.md
+          - install-pmm/install-pmm-server/virtual/login_UI.md
+          - install-pmm/install-pmm-server/virtual/remove_virtual_machine.md
+        - Install PMM with AWS marketplace:
+          - install-pmm/install-pmm-server/aws/aws.md
+          - install-pmm/install-pmm-server/aws/limit_access.md
+          - install-pmm/install-pmm-server/aws/settings.md
+          - install-pmm/install-pmm-server/aws/run.md
+          - install-pmm/install-pmm-server/aws/back_pmm_server.md
+          - install-pmm/install-pmm-server/aws/restore_pmm_server.md
+          - install-pmm/install-pmm-server/aws/remove_pmm_server.md
+        - Install PMM with DBaaS:
+          - install-pmm/install-pmm-server/dbaas/dbaas.md
+          - install-pmm/install-pmm-server/dbaas/activate_dbaas.md
+          - install-pmm/install-pmm-server/dbaas/create_k8s_cluster.md
+          - install-pmm/install-pmm-server/dbaas/del_cluster.md
+          - install-pmm/install-pmm-server/dbaas/run_as_docker.md
+          - install-pmm/install-pmm-server/dbaas/expose_clusters_for external_clients.md
+        - install-pmm/install-pmm-server/easy-install.md
+      - Install PMM client:
+        - install-pmm/install-pmm-client/index.md
+        - install-pmm/install-pmm-client/prerequisites.md 
+        - install-pmm/install-pmm-client/docker.md
+        - install-pmm/install-pmm-client/binary_package.md          
+        - install-pmm/install-pmm-client/package_manager.md
+      - Register client node: install-pmm/register-client-node/index.md 
+      - Add services:      
+        - install-pmm/add-services/index.md
+        - install-pmm/add-services/aws.md
+        - install-pmm/add-services/azure.md
+        - install-pmm/add-services/external.md
+        - install-pmm/add-services/google.md
+        - install-pmm/add-services/haproxy.md
+        - install-pmm/add-services/linux.md
+        - install-pmm/add-services/mongodb.md
+        - install-pmm/add-services/mysql.md
+        - install-pmm/add-services/postgresql.md
+        - install-pmm/add-services/proxysql.md
+        - install-pmm/add-services/remote.md
+  - Configure PMM:      
+      - configure-pmm/configure.md
+      - configure-pmm/metrics_res.md
+      - configure-pmm/advanced_settings.md
+      - configure-pmm/public-address.md
+      - configure-pmm/ssh.md
+      - configure-pmm/alertmanager.md
+      - configure-pmm/percona_platform.md
+  - Upgrade PMM:
+      - pmm-upgrade/index.md
+      - pmm-upgrade/ui_upgrade.md
+      - pmm-upgrade/upgrade_aws.md    
+      - pmm-upgrade/upgrade_docker.md
+      - pmm-upgrade/upgrade_agent.md
+      - pmm-upgrade/upgrade_from_pmm_1.md
+  - Uninstall PMM:
+      - uninstall-pmm/index.md
+      - uninstall-pmm/uninstall_docker.md
+      - uninstall-pmm/uninstall_package_manager.md
+      - uninstall-pmm/unregister_client.md
```

Comment:
I would reorder sections in the following way:
- Install PMM:
- Uninstall PMM:
- Upgrade PMM:
- Configure PMM:
- Manage PMM: